### PR TITLE
Added a new process

### PR DESCRIPTION
**Pull request by CodeAndManage888**
In determining the data and computing for Fahrenheit and MPH.

### CODE REVIEW

File: 00E028.py [hunk 1]
```diff
@@ -29,39 +30,61 @@
...
+      icheck = -1
 #--------------------------------------------------------------
 print("Thank you for using this app.")
 #**************************************************************
 # Open Items:
 # 1.) The program will start with *C and KPH. Still need to check
 #     the exact formula for WCI if the data is for *F and MPH.
 # 2.) Still need to validate the final computed value.
-# 
+#     WCI = 35.74 + 0.6215Ta * (0.4275Ta - 35.75) * V^0.16
```
Comment (boredmanager888) [inline 00E028.py:89]: Need further details around this notes.

File: 00E028.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 # speeds in miles per hour.                                   *
```
Comment (boredmanager888) [inline 00E028.py:15]: Status should now be in testing.

File: 00E028.py (hunk 1 above)
Comment (CodeAndManage888) [reply 00E028.py:89]: Will add details in the new iteration.

File: 00E028.py (hunk 2 above)
Comment (CodeAndManage888) [reply 00E028.py:15]: Will update the status in the next iteration. Program needs to be tested first.

**Review by boredmanager888 — APPROVED** [review]

**Comment by boredmanager888** [comment]
Merged approved.

**Pull request merged**